lib/simulator-xcode-7.js

```diff
@@ -47,6 +47,14 @@ class SimulatorXcode7 extends SimulatorXcode6 {
       'iPhone 6s Plus (9.2)': 'iPhone 6s Plus (9.2) [',
     };
   }
+
+  async openUrl (url) {
```

Comment:
If this is only in Xcode 7, can we have some sort of thing in the 6 class so we won't get `undefined is not a function` errors? Just throw an error that makes sense?